gst/nnstreamer/tensor_demux/gsttensordemux.c

```diff
@@ -462,7 +462,7 @@ gst_tensor_demux_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
     srcpad = gst_tensor_demux_get_tensor_pad (tensor_demux, &created, i);
 
     outbuf = gst_buffer_new ();
-    mem = gst_buffer_peek_memory (buf, i);
+    mem = gst_buffer_get_memory (buf, i);
```

Comment:
refer: https://developer.gnome.org/gstreamer/stable/gstreamer-GstBuffer.html#gst-buffer-get-memory

You need gst_memory_unref()

Reply:
gst_buffer_append_memory() will take the ownership of mem-block, so it is necessary to increase ref count of memory before calling the gst_buffer_unref (buf)

Reply:
Oh.. great!